src/app/record-ide/components/record-ide.component.ts

```diff
@@ -102,16 +101,13 @@ export class RecordIDEComponent implements OnInit {
     this.project$ = this.projectApolloService.getProjectById(projectId);
     this.subscriptions$.push(this.project$.subscribe((project) => {
       this.project = project;
-      if (this.firstVisit) {
-        this.runRecordIde();
-        this.firstVisit = false;
-      }
+      this.runRecordIde(true);
     }));
     return this.project$.pipe(first());
   }
 
-  runRecordIde() {
-    if (this.code.indexOf("import easteregg") != -1) {
+  runRecordIde(firstVisit: bool = false) {
```

Comment:
Is there a specific reason why we use bool and not boolean? I think we use boolean through the application.

Reply:
no, i will change it